types/truffle-global.d.ts

```diff
@@ -1,8 +0,0 @@
-import * as Web3 from 'web3';
-
-declare global {
-  const web3: Web3;
-  const artifacts: SignHashArtifacts;
-  const assert: Chai.AssertStatic;
-  const contract: ContractContextDefinition;
-}
```

Comment:
I'm not sure why have you removed that.

Reply:
There are two problems with this approach:

Not in all contexts these variables are defined (e.g. seed)
WebStorm does not recognize them (pretty lousy argument, but I heavily rely on this IDE)